evaluator/autoware_control_evaluator/src/control_evaluator_node.cpp

```diff
@@ -66,28 +60,7 @@ void ControlEvaluatorNode::getRouteData()
   }
 }
 
-void ControlEvaluatorNode::onDiagnostics(const DiagnosticArray::ConstSharedPtr diag_msg)
-{
-  // add target diagnostics to the queue and remove old ones
-  for (const auto & function : target_functions_) {
-    autoware::evaluator_utils::updateDiagnosticQueue(*diag_msg, function, now(), diag_queue_);
-  }
-}
-
-DiagnosticStatus ControlEvaluatorNode::generateAEBDiagnosticStatus(const DiagnosticStatus & diag)
-{
-  DiagnosticStatus status;
-  status.level = status.OK;
-  status.name = diag.name;
-  diagnostic_msgs::msg::KeyValue key_value;
-  key_value.key = "decision";
-  const bool is_emergency_brake = (diag.level == DiagnosticStatus::ERROR);
-  key_value.value = (is_emergency_brake) ? "deceleration" : "none";
-  status.values.push_back(key_value);
-  return status;
-}
-
-DiagnosticStatus ControlEvaluatorNode::generateLaneletDiagnosticStatus(const Pose & ego_pose) const
```

Comment:
If this method is deleted. AEB needs to generate a metric for both "none" and "decelerateion"/ "brake"